src/components/FormValidator.js

```diff
@@ -84,4 +85,12 @@ export default class FormValidator {
     });
     this._setEventListiners();
   }
+
+  resetValidation() {
+    const inputList = this._element.querySelectorAll(".modal__input");
```

Comment:
`    const inputList = this._element.querySelectorAll(".modal__input");
`

You've already found `this._inputEls` in line 59. Please, use it here

`Class fields` (the so-called **class variables** with `this`) are accessible inside any class method. You shouldn’t pass **class fields** into the call of any method or search them again because you can use class fields inside all methods.